### PR TITLE
Adding bridges to site tests and configuration docs

### PR DESCRIPTION
**Pull request by yadudoc**
# Description

This PR adds support for site testing on Bridges and adds a tested configuration to the docs.

This also contains updates to conda_init documenting how to get conda env on bridges

## Type of change

Please check the options below:

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Code maintentance/cleanup
- [x] Updates to existing or new tests

### CODE REVIEW

**Comment by ZhuozhaoLi** [comment]
there are some flake8 errors

File: parsl/configs/bridges.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+of Bridges at PSC. Each job submitted to the scheduler will request 2 nodes for 10 minutes.
+"""
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            label='Bridges_HTEX_multinode',
+            max_workers=1,
+            provider=SlurmProvider(
+                'YOUR_PARTITION_NAME',  # Specify Partition / QOS, for eg. RM-small
```
Comment (danielskatz) [inline parsl/configs/bridges.py:16]: this shouldn't have "partition=" before it?

**Comment by danielskatz** [comment]
`channel=LocalChannel()` is not needed in the provider?

File: parsl/configs/bridges.py (hunk 1 above)
Comment (yadudoc) [reply parsl/configs/bridges.py:16]: It's a positional arg, so not necessary.

Comment (danielskatz) [reply parsl/configs/bridges.py:16]: it seems like it would be cleaner/clearer to have it

Comment (danielskatz) [reply parsl/configs/bridges.py:16]: but I'll approve anyhow

**Pull request merged**